Assets/MRTK/Core/Inspectors/MixedRealityStandardShaderGUI.cs

```diff
@@ -470,6 +476,13 @@ protected void MainMapOptions(MaterialEditor materialEditor, Material material)
                 materialEditor.ShaderProperty(triplanarMappingBlendSharpness, Styles.triplanarMappingBlendSharpness, 2);
             }
 
+            // SSAA implementation based off this article: https://medium.com/@bgolus/sharper-mipmapping-using-shader-based-supersampling-ed7aadb47bec
+            materialEditor.ShaderProperty(enableSSAA, Styles.enableSSAA);
+            if (PropertyEnabled(enableSSAA))
```

Comment:
nit: we might want to grey out (and disable) ` materialEditor.ShaderProperty(enableTriplanarMapping, Styles.enableTriplanarMapping);` option above if `PropertyEnabled(enableSSAA)` is true since the two are mutually exclusive in the shader.